tests/backup/backup_test.go

```diff
@@ -303,6 +309,232 @@ func TeardownForTestcase(contexts []*scheduler.Context, providers []string, Clou
 
 }
 
+var _ = Describe("{ShareBackupWithUsersAndGroups}", func() {
+	numberOfUsers := 30
+	numberOfGroups := 3
+	groupSize := 10
+	numberOfBackups := 9
+	users := make([]string, 0)
+	groups := make([]string, 0)
+	backupNames := make([]string, 0)
+	var contexts []*scheduler.Context
+	labelSelectors := make(map[string]string)
+	var backupLocationUID string
+	var customBucketName string
+	var cloudCredUID string
+	var CloudCredUidList []string
+	var appContexts []*scheduler.Context
+	var bkpNamespaces []string
+	var clusterUid string
+	var clusterStatus api.ClusterInfo_StatusInfo_Status
+	var customBackupLocationName string
+	var credName string
+	bkpNamespaces = make([]string, 0)
+	providers := getProviders()
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("Backup: Share large number of backups",
```

Comment:
Avoid using space and special characters as this string is used to creat a test log file,
Use the testname , so by seeing file name we can identify the test

Reply:
Fixed it. Please refer [Aetos Dashboard](http://aetos.pwx.purestorage.com/resultSet/testSetID/74560)